packages/augur-sdk/src/state/create-api.ts

```diff
@@ -157,6 +167,10 @@ export async function createServer(config: SDKConfiguration, client?: Augur, acc
     ...config
   };
 
+  if (!(config.ethereum && config.ethereum.http)) {
```

Comment:
This isn't actually true -- this is only true if running in server mode (and thus this should be checked outside of createServer).

e.g. for augur in the UI all we need to do is pass in a Provider.

Reply:
I moved this check to where we actually use `config.ethereum.http`, and only when we don't provide a provider.